doc/policy/packages.md

```diff
@@ -18,7 +18,7 @@ tip or some preceding transaction in the package.
 
 The following rules are enforced for all packages:
 
-* Packages cannot exceed `MAX_PACKAGE_COUNT=25` count and `MAX_PACKAGE_SIZE=101KvB` total size
+* Packages cannot exceed `MAX_PACKAGE_COUNT=25` count and `MAX_PACKAGE_KWU=404` total weight
    (#20833)
 
    - *Rationale*: This is already enforced as mempool ancestor/descendant limits. If
```

Comment:
Ah hold on, the lack of rounding is more accurate but it does mean that this limit is effectively a few Wu smaller than the default ancestor size limit. (https://github.com/bitcoin/bitcoin/pull/22097#issuecomment-859901930 and https://github.com/bitcoin/bitcoin/pull/22097#issuecomment-862627568) so this statement is untrue.

I think having this limit denominated in Wu is good and it should be distinct from the ancestor package limits anyway. But I agree with those comments that maybe it's best to not make this limit more restrictive than normal ancestor limits.

We can bake in the rounding so this limit is 404000Wu + MAX_PACKAGE_COUNT * (WITNESS_SCALE_FACTOR -1) = 404075Wu.

Also I think the primary Rationale (if you could edit this bullet point please) should be "We want package size to be as small as possible to limit resources needed to validate packages. However, we want to make sure that the limit allows for expected ancestor packages."

Reply:
Fixed up as suggested, and fixed some inverted static asserts.

I thought about calling it 405KWU and calling it a day, but I think that would make code history more difficult, so instead stayed precisely as calculated.

Reply:
Oops, those comments are incorrect. The rounding *does* mean that this weight limit across multiple transactions is different from the ancestor size limits, but it makes the package limits *less* restrictive, not more restrictive. That's totally fine. No change needed, we can just keep 404,000Wu.